css/style.css

```diff
@@ -60,7 +61,6 @@ form {
 #email {
   padding: 10px;
   margin-top: 20px;
-  border: none;
```

Comment:
There is a duplicate of `border` in this property, it best practice not to duplicate values.

Reply:
This is a nice comment however, you did not clearly tell him what you want him to do after pointing out that there is a duplicate and giving him a 'why' to not duplicate values. It would be better to tell the author what you want them to do from your comment. For example:

*There is a duplicate of border in this property, it is best practice not to duplicate values. Kindly ensure that you remove the duplicate*